system-x/services/hawtio/src/main/java/software/tnb/hawtio/service/Hawtio.java

```diff
@@ -0,0 +1,19 @@
+package software.tnb.hawtio.service;
+
+import software.tnb.common.account.NoAccount;
+import software.tnb.common.service.Service;
+import software.tnb.hawtio.client.HawtioClient;
+import software.tnb.hawtio.validation.HawtioValidation;
+
+public abstract class Hawtio extends Service<NoAccount, HawtioClient, HawtioValidation> {
+
+    @Override
+    public abstract HawtioClient client();
```

Comment:
I think this may not be necessary and the clients can be initialized with simple `client = new Local/OpenshiftHawtioClient(getHawtioUrl());` in `openResources()` method?